pkg/api/admin/openshiftcluster.go

```diff
@@ -46,6 +46,7 @@ type OpenShiftClusterProperties struct {
 	NetworkProfile                  NetworkProfile          `json:"networkProfile,omitempty"`
 	MasterProfile                   MasterProfile           `json:"masterProfile,omitempty"`
 	WorkerProfiles                  []WorkerProfile         `json:"workerProfiles,omitempty"`
+	WorkerProfilesStatus            []WorkerProfile         `json:"workerProfilesStatus,omitempty"`
```

Comment:
can we add a comment about this property. in particular, how is it different than the one above it.